doc/nrf/protocols/matter/getting_started/adding_bt_services.rst

```diff
@@ -0,0 +1,69 @@
+* Using `Bluetooth SIG's Assigned Numbers`_ documentation with predefined characteristics to create your custom implementation.
+* Writing the custom Bluetooth LE service from scratch.
+
+This guide describes the first case and uses :ref:`nus_service_readme` as an example.
+
+.. _ug_matter_device_adding_bt_services_ble_arbiter:
+
+Advertising arbiter for Bluetooth LE services
+*********************************************
+
+The Bluetooth LE Arbiter is a class implemented in the Zephyr platform within the Matter SDK.
+It enables easier coexistence of application components that want to advertise their Bluetooth LE services, for example during :ref:`ug_matter_network_topologies_commissioning_stages_discovery`.
+
+When several application services are active, they can define a request to the Arbiter with the desired priority.
+If the service with the highest priority stops advertising, the Arbiter automatically selects the next service in the queue.
+If the service that requests advertising has a higher priority than the service that is running when the request is submitted, the advertising is restarted using parameters defined in the new request.
+
+In the Matter implementation in the |NCS|, the following services are configured by default for all Matter applications:
+
+* Commissioning for Matter (priority ``0``)
+* DFU over Bluetooth LE (priority ``uint8 max``)
+
+Lower number has a higher priority.
+If you implement a custom Bluetooth LE service, you need to prioritize it with the Bluetooth LE Arbiter.
+
+For more information about what the class offers, see the `Bluetooth LE Arbiter's header file`_.
+
+.. _ug_matter_device_adding_bt_services_adding:
+
+Adding support for a Bluetooth LE service
+*****************************************
+
+To add support for a Bluetooth LE service implemented in the |NCS|, complete the following steps:
+
+1. Check the documentation of the :ref:`Bluetooth service <lib_bluetooth_services>` for its characteristics and functions.
+#. Add the application code that instantiates and fills the `Bluetooth LE Arbiter's Request struct`_, including its ``priority`` field.
+#. Add the application code that calls the `Bluetooth LE Arbiter's InsertRequest function`_, with the reference to the ``Request`` structure, when the service needs to be advertised.
+   This adds the service to the Arbiter queue.
+
+.. _ug_matter_device_adding_bt_services_example:
+
+Example: Bluetooth LE with Nordic UART Service implementation
+=============================================================
+
+The :ref:`Matter door lock sample` provides an optional configuration of :ref:`matter_lock_sample_ble_nus`, which you can use as an example implementation of a Bluetooth LE service.
+Once enabled, you can use this configuration to declare NUS commands specific to a Matter sample and use them to control the device remotely through Bluetooth LE.
+
+The following steps mirror the implementation steps with the NUS as example:
```

Comment:
Are the following lines steps? I guess the steps should start with `Do something...`.